src/navigators/DrawerNavigator.js

```diff
@@ -82,14 +86,17 @@ const DrawerNavigator = (
     drawerPosition,
     useNativeAnimations,
     drawerBackgroundColor,
+    drawerOpenRoute,
```

Comment:
With this change these three properties are mandatory if you override the `config`. This is due to `config` overriding the `DefaultDrawerConfig`. Was that intentional?